data/fields/seamark/beacon_isolated_danger/shape.json

```diff
@@ -1,5 +1,12 @@
 {
     "key": "seamark:beacon_isolated_danger:shape",
     "type": "combo",
-    "label": "Shape"
+    "label": "Shape",
+    "strings": {
+        "options": {
+            "pile": "Pile",
+            "stake": "Stake",
+            "tower": "Tower"
```

Comment:
`seamark:beacon_*:shape` is a standard key, so I would suggest defining all the possible options, not just these 3.

[Here's how the preset is defined in a fork of iD](https://github.com/k-yle/id-tagging-schema/blob/f202907/data/fields/seamark/beacon_lateral/shape.json#L6-L14) (see [here](https://kyle.kiwi/iD/#id=n10251795794&overlays=openseamap-overlay&map=18/-33.83677/151.27212) for a preview)

I would also suggest updating the field for `seamark:beacon_lateral:shape`, so that it references the same options. [Example of the code here](https://github.com/k-yle/id-tagging-schema/blob/f202907/data/fields/seamark/beacon_cardinal/shape.json#L5)

Reply:
@k-yle

> I would also suggest updating the field for seamark:beacon_lateral:shape, so that it references the same options.

I've read the schema-builder [section](https://github.com/ideditor/schema-builder?tab=readme-ov-file#stringscrossreference) about cross referencing fields between presets. How do I reference fields from a different folder?

Reply:
@kjonosm [here's an example](https://github.com/k-yle/id-tagging-schema/blob/f202907/data/fields/seamark/beacon_cardinal/shape.json#L5)